specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```
@@ -479,6 +479,13 @@
           {
             "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
           },
+          {
+            "name": "location",
```

nit, I think you can use this instead

```suggestion
          {
            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
          },
```